File: Modul 3/SOAL-5/PRAK305_2410817220015_Ashma_Salimah.py
a = int(input(""))
hari = a // 86400
detik = a % 86400
jam = detik // 3600
detik %= 3600
menit = detik // 60
detik2 = detik % 60
if hari > 0 : print(f"{int(hari)} hari {jam:02}:{menit:02}:{detik2:02}")
else : print(f"{jam:02}:{menit:02}:{detik2:02}")

File: Modul 3/SOAL-5/PRAK305_2410817220015_Ashma_Salimah.c
#include <stdio.h>
int main () {
    int nilai, hari, jam, menit, detik;
    scanf("%d", &nilai);
    hari = nilai / 86400;
    jam = (nilai % 86400) / 3600;
    menit = (nilai % 3600) / 60;
    detik = nilai % 60;
    if (hari > 0) {printf("%d hari %02d:%02d:%02d", hari, jam, menit, detik);}
    else {printf("%02d:%02d:%02d", jam, menit, detik);}
    return 0;
}